cluster-operator/src/main/java/io/strimzi/operator/cluster/Main.java

```diff
@@ -121,11 +117,11 @@ static CompositeFuture run(Vertx vertx, KubernetesClient client, PlatformFeature
                 networkPolicyOperator, podDisruptionBudgetOperator, resourceOperatorSupplier, config.versions(), config.getImagePullPolicy());
 
         KafkaConnectS2IAssemblyOperator kafkaConnectS2IClusterOperations = null;
-        if (pfa.isOpenshift()) {
+        if (pfa.isBuilds() && pfa.isApps() && pfa.isImages()) {
```

Comment:
I think the method names starting `supports...` or `has...` would make this read slightly more naturally.